test.tsx

```diff
@@ -31,11 +30,13 @@ const FooBarBaz = {
   },
   controller: FooController,
   template: `
-    <span>{{ $ctrl.foo }}</span>
-    <span>{{ $ctrl.fooBar*12 }}</span>
-    <span>{{ $ctrl.fooService.add42($ctrl.fooBar) }}</span>
-    <div ng-click="$ctrl.onClick()">Click Me</div>
-    <ng-transclude></ng-transclude>
+    <span id="foobarbaz">
```

Comment:
With this PR, do component templates now need to be wrapped in an enclosing element?